src/common/store/cache/users/sagas.js

```diff
@@ -217,16 +217,23 @@ function* watchSyncLocalStorageUser() {
   yield takeEvery(cacheActions.UPDATE, syncLocalStorageUser)
 }
 
+// Adjusts a user's field in the cache by specifying an update as a delta.
+// The cache respects the delta and merges the objects adding the field values
 export function* adjustUserField({ user, fieldName, delta }) {
```

Comment:
First important change. When following users, we call "adjustUserField" on our account to increment our "followee_count" by 1 for each user we have followed.

This saga is buggy. It effectively tries to be redux (pull state, update value w/ delta, write state). The problem is that between pulling state and updating state, because this is a saga, other stuff could have happened and that initial pulled state is not correct. This needs to be done synchronously, so I've added that to the reducer.